OktaLogger.podspec

```diff
@@ -54,6 +55,14 @@ Pod::Spec.new do |s|
         'Sources/OktaLogger/AppCenterLogger',
         'Sources/OktaLogger/InstabugLogger'
       ]
+      core.ios.dependency 'OktaLogger/Analytics'
```

Comment:
Is it really should be part of Logger Core?

Reply:
Core should not have any sub-spec dependencies as it is base implementation

Reply:
we don't need it I had changed the implementation